common-api/src/main/java/org/pmiops/workbench/cdr/model/Concept.java

```diff
@@ -184,6 +186,19 @@ public Concept prevalence(float prevalence) {
         return this;
     }
 
+    @Column(name = "synonyms")
+    public String getSynonymsStr() {
+        return synonymsStr;
+    }
+
+    public void setSynonymsStr(String synonymsStr) { this.synonymsStr = synonymsStr; }
+
+    public Concept synonymsStr(String synonymsStr) {
+        this.synonymsStr = synonymsStr;
+        return this;
+    }
+
+
```

Comment:
What is the plan for the synonm relationship? Remove it eventually?

Reply:
Duh. I see. Nice . You use this and set the synonms array and made it transient. Looks good.

Reply:
Looks good